lib/crypto/doc/src/crypto.xml

```diff
@@ -835,6 +835,8 @@ end
 	<p>For decryption, set the <c>EncryptFlag</c> to <c>false</c> and put the tag to be checked
 	in the argument	<c>TagOrTagLength</c>.
 	</p>
+  <p>The <c>AAD</c> argument represents the Additional Authentication Data. It's optional. If it's
```

Comment:
I suggest that we should clarify this a little further. I suggest:

"AAD (Additional Authenticated data)  is plaintext that will not be encrypted, but will be covered by authenticity protection, and may be provided but can be the empty binary and in that case a plain AE (Authenticated Encryption) is performed instead of the AEAD (Authenticated Encryption with Associated Data). This function only supports ciphers
that can be used in both with and without AAD. "

What do you think? See also https://en.wikipedia.org/wiki/Authenticated_encryption

Reply:
@IngelaAndin I like that, thanks for expanding on the doc! I updated the PR with your suggestion 🙃